packages/Uploader/README.md

```diff
@@ -1,3 +1,58 @@
+| a11yText                                                                                                     | string  | false    | null              | Accessible message for the input[type="file"].                                                                |
+| okFileTypes                                                                                                  | arrayOf | false    | ["*/*"]           | An array of string describing the allowed file types for the uploader ex. ["image/*", ".pdf", ".doc", "docx"] |
+| reference: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#Unique_file_type_specifiers. |
+| canChooseMultiple                                                                                            | bool    | false    | true              | When false the uploader only accept one file per upload.                                                      |
+| children                                                                                                     | node    | true     | -                 | children nodes                                                                                                |
+| defaultIsDisabled                                                                                            | bool    | false    | false             | initial disable state for the uploader                                                                        |
+| endpoint                                                                                                     | string  | true     | -                 | The url that will be use to upload the files ex. https://yourEndPointURL.com                                  |
+| hasAutoUpload                                                                                                | bool    | false    | true              | On true will upload the file as soon they are selected or dropped                                             |
+| isBodyDroppable                                                                                              | bool    | false    | true              | When true the user will be able to drop files at any part of the document.body triggering                     |
+| the upload listeners. On false will only received files if they are drop exactly on the FileInput area.      |
+| maxFileSize                                                                                                  | number  | false    | oneMebibyte \* 10 | Size in Mebibytes which is use for comparing each file that will be upload                                    |
+
+    you can make use of Uploader.convertUnitsToMebibytes() for easily convert units to Mebibyes
+    ex.  Uploader.convertUnitsToMebibytes(4) => 4194304 (close to 4MB);|
+
```

Comment:
This one is broken (see the markdown preview of the file)

Reply:
yeah this one and the listboxbrowser are broken something on the comments are breaking markdown, the comments should need to move out of the source. we are cleaning the markdown in a parallel branch with Karen and Kaan.